src/components/SettingsComponents/ChangePwModal/ChangePwModal.tsx

```diff
@@ -35,7 +35,7 @@ const ChangePwModal: FC<ChangePwModalProps> = (props) => {
       <h3 className='font-bold'>{t('settings.change_pw')}</h3>
       {isLoading && <LoadingSpinner />}
       {!isLoading && (
-        <div className='my-5 flex flex-col justify-center text-center items-center'>
+        <div className='mt-5 flex flex-col justify-center text-center items-center'>
```

Comment:
With the change on pb-10 that would look weird

depending on what we are doing with pb-10, I would revert that to it's previous state

```suggestion
        <div className='my-5 flex flex-col justify-center text-center items-center'>
```